app/src/main/java/org/mozilla/fenix/components/Performance.kt

```diff
@@ -0,0 +1,11 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.components
+
+import org.mozilla.fenix.utils.StartupTaskManager
+
+class Performance {
```

Comment:
nit: there's a namespace collision between this and the `object Performance`: should be call this `PerformanceComponent` instead? I don't think anyone will ever need to refer to it by name.